mac_build.sh

```diff
@@ -80,17 +90,22 @@ export VORBISENC_CFLAGS="-I$(pwd)/libvorbis-$VORBISVERSION/include"
 export VORBISENC_LIBS="$(pwd)/libvorbis-$VORBISVERSION/lib/libvorbisenc.la"
 export OPUS_CFLAGS="-I$(pwd)/opus-$OPUSVERSION/include"
 export OPUS_LIBS="$(pwd)/opus-$OPUSVERSION/libopus.la"
-export LAME_CFLAGS="-I$(pwd)/lame-$LAMEVERSION/include"
-export LAME_LIBS="$(pwd)/lame-$LAMEVERSION/libmp3lame/.libs/libmp3lame.la"
-export MPG123_CFLAGS="-I$(pwd)/mpg123-$MPG123VERSION/src/libmpg123 $LAME_CFLAGS"
-export MPG123_LIBS="$(pwd)/mpg123-$MPG123VERSION/src/libmpg123/libmpg123.la $LAME_LIBS"
+export MPEG_CFLAGS="-I$(pwd)/lame-$LAMEVERSION/include"
+export MPEG_LIBS="$(pwd)/lame-$LAMEVERSION/libmp3lame/.libs/libmp3lame.la"
```

Comment:
According to [the build log](https://github.com/bastibe/libsndfile-binaries/actions/runs/4082021218/jobs/7035934334#step:3:5253), the output directory might be `libmp3lame/libmp3lame.la`. It looks like only the `.a` goes to `.lib`.

Reply:
Yeah I think cmake requires linking to .a files, not .la actually. So this is almost working, just need a static library for opus to link against